apps/meteor/jest.config.ts

```diff
@@ -33,6 +33,7 @@ const config: Config = {
 				'<rootDir>/app/livechat/server/business-hour/**/*.spec.ts?(x)',
 				'<rootDir>/app/livechat/server/api/**/*.spec.ts',
 				'<rootDir>/ee/app/authorization/server/validateUserRoles.spec.ts',
+				'<rootDir>/app/cloud/server/functions/supportedVersionsToken/**.spec.ts',
```

Comment:
maybe for another PR but, how can we make this not needed? like adding a new test unit file without having to worry (too much) about it being picked up or not.

Reply:
sure